manager/src/main/java/org/apache/hertzbeat/manager/service/impl/NoticeConfigServiceImpl.java

```diff
@@ -238,6 +238,11 @@ public NoticeRule getNoticeRulesById(Long ruleId) {
         return noticeRuleDao.getReferenceById(ruleId);
```

Comment:
Suggest we can modify this method instead of create a new method

Reply:
okay.